README.md

```diff
@@ -225,3 +225,82 @@ Example:
     ```
     pytest -s ecs/mxnet/training/test_ecs_mxnet_training.py::test_ecs_mxnet_training_dgl_cpu
     ```
+
+7. To run sagemaker Local tests, Launch a cpu or gpu instance based with latest deep learning AMI.
+    a) clone your github branch with changes and run the following commands
```

Comment:
This wording is a bit confusing. Are we assuming the users of this will be using a fork as opposed to the main repo? I might change to: "Once connected to the instance, run the following commands to clone this repository:"